### DISCUSSION TITLE
It seems these functions in pacing.rs are not called

### DISCUSSION THREAD

**Discussion by SoonyangZhang** [Tencent/tquic#273] [General]
The conn sends packets with intervals determined by pacing rate, BBR can work normally.

**Comment by glime-ai** [comment]
Yes, BBR (Bottleneck Bandwidth and Round-trip propagation time) congestion control can function effectively by pacing the transmission of packets based on a calculated pacing rate. This pacing rate is dynamically adjusted to match the estimated bottleneck bandwidth (BtlBw) and the round-trip time (RTT) of the network path. BBR aims to optimize both throughput and latency by controlling the rate at which data is sent, ensuring that the data flow matches the available network capacity without causing excessive delay or packet loss.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=44520fce49ba43798485a628d6928e22&utm_medium=github).

**Comment by iyangsj** [comment]
@SoonyangZhang Pacing is currently in development and will be fully supported in an upcoming version.
